lib/vagrant-nfs_guest/guests/debian/cap/nfs_server.rb

```diff
@@ -0,0 +1,22 @@
+module VagrantPlugins
+  module SyncedFolderNFSGuest
+    module GuestDebian
+      module Cap
+        class NFSServer
+          def self.nfs_test_command(env)
+            "test -x /usr/sbin/exportfs"
```

Comment:
The Linux guest uses `which exportfs`, but the test seems to be run as a non-root user, and `exportfs` is not in non-user `PATH` on Debian.